### ISSUE TITLE
rspotify fails to build with blocking feature

### ISSUE THREAD

**Issue by cosmoduff** [ramsayleung/rspotify#87]
Reqwest has blocking set as an optional feature and is off by default. When trying to build rspotify with the blocking feature you get error messages similar to the following:
```
error[E0432]: unresolved import `reqwest::blocking`
 --> src/blocking/client.rs:5:14
  |
5 | use reqwest::blocking::Client;
  |              ^^^^^^^^ could not find `blocking` in `reqwest`

error[E0432]: unresolved import `reqwest::blocking`
 --> src/blocking/oauth2.rs:6:14
  |
6 | use reqwest::blocking::Client;
  |              ^^^^^^^^ could not find `blocking` in `reqwest`

error[E0433]: failed to resolve: could not find `blocking` in `reqwest`
  --> src/blocking/client.rs:65:21
   |
65 | impl From<&reqwest::blocking::Response> for ApiError {
   |                     ^^^^^^^^ could not find `blocking` in `reqwest`

error[E0433]: failed to resolve: could not find `blocking` in `reqwest`
  --> src/blocking/client.rs:66:33
   |
66 |     fn from(response: &reqwest::blocking::Response) -> Self {
   |                                 ^^^^^^^^ could not find `blocking` in `reqwest`

error[E0599]: no method named `request` found for type `blocking::client::CLIENT` in the current scope
   --> src/blocking/client.rs:153:34
    |
38  | / lazy_static! {
39  | |     /// HTTP Client
40  | |     pub static ref CLIENT: Client = Client::new();
41  | | }
    | |_- method `request` not found for this
...
153 |               let builder = CLIENT.request(method, &url.into_owned()).headers(headers);
    |                                    ^^^^^^^ method not found in `blocking::client::CLIENT`
    |
    = note: this error originates in a macro outside of the current crate (in Nightly builds, run with -Z external-macro-backtrace for more info)

```

**Comment by sputnick1124** [comment]
What git-sha are you trying to build? What version of rust?
Currently the master branch (https://github.com/ramsayleung/rspotify/commit/96fe932724c48e47ba8831790f43aacc405a5a36) builds for me with `cargo build --features=blocking`.
```bash
$ rustc --version
rustc 1.42.0 (b8cedc004 2020-03-09)
```

**Comment by cosmoduff** [comment]
This was fixed by pull #88 and can be closed.

**Comment by ramsayleung** [comment]
Yes, @cosmoduff is right, this problem was fixed :)
